core/primitives/src/views.rs

```diff
@@ -1715,6 +1715,11 @@ pub enum TxExecutionStatus {
     None,
     /// Transaction is included into the block. The block may be not finalised yet
     Included,
+    /// Transaction is included into the block +
+    /// All the transaction receipts finished their execution.
+    /// The corresponding blocks for tx and each receipt may be not finalised yet
+    #[default]
+    ExecutedOptimistic,
```

Comment:
This option is default for all methods, so all the parsers will pick it for `broadcast_tx_commit`, `tx`, `EXPERIMENTAL_tx_status`.

Reply:
@telezhnaya Please, move this enum variant to be the last on the list of enum options or provide explicit tags to the enum values since if anyone ever borsh-serialized `TxExecutionStatus` and stored it somewhere, they will get a wrong value on deserialization! (`IncludedFinal` -> `ExecutedOptimistic`)